src/Effects.js

```diff
@@ -237,34 +329,38 @@ module.exports = class Effects {
 
     this.raining_tacos = {
       tacos: [],
-      size: 50,
       init: function () {
-        for (let i = 0; i < 20; i++) {
+        for (let i = 0; i < 10; i++) {
           this.tacos.push({
             x: randomNumber(20, 380),
-            y: randomNumber(20, 380),
+            y: randomNumber(-400, 0),
             rot: randomNumber(0, 359),
-            speed: randomNumber(2, 5),
+            speed: 3,
+            size: 5,
           });
         }
+        this.image = p5.createGraphics(70, 50);
+        this.image.pixelDensity(1);
+        this.image.scale(4,4);
+        drawTaco(this.image.drawingContext);
```

Comment:
:+1: Looking forward to seeing what a difference this makes in performance!

Let's make sure we test this in Safari, since it has known issues with `createGraphics`.  I did investigate a little and I don't see a way to inject our own offscreen canvas context into p5 for drawing - I suspect this will be okay just happening on init with a fairly small canvas, but want to validate before we ship the associated blocks.